airflow/providers/amazon/aws/executors/ecs/ecs_executor.py

```diff
@@ -470,3 +482,44 @@ def get_container(self, container_list):
             if container["name"] == self.container_name:
                 return container
         raise KeyError(f"No such container found by container name: {self.container_name}")
+
+    def try_adopt_task_instances(self, tis: Sequence[TaskInstance]) -> Sequence[TaskInstance]:
```

Comment:
I think the commits might have gotten mixed up here. This PR is doing 2 unrelated things it seems.

Reply:
Yeah, I had some git issues in another PR I was working on at the same time and thought I untangled them, but it looks like I didn't sort out this branch.

Reply:
This is fixed now, sorry for the confusion.